src/tokens/transformation/skia/index.ts

```diff
@@ -131,7 +129,7 @@ const templates = [
 for (const templateName of templates) {
   StyleDictionary.registerFormat({
     name: `skia/${templateName}`,
-    formatter: ({ dictionary, options, file }) => {
+    format: async ({ dictionary, options, file }) => {
```

Comment:
this doesn't seem like its actually async?

Reply:
Blaming these on the automigration tool as I don't recall doing this myself. :-)